server/datasource.ts

```diff
@@ -1,18 +1,55 @@
-import 'reflect-metadata';
+import fs from 'fs';
+import * as process from 'process';
```

Comment:
You don't need to import that, `process` is available in the global context by default.
```suggestion
```